shuup/configuration.py

```diff
@@ -118,6 +121,8 @@ def _get_configuration_from_db(shop):
     configuration = {}
     for conf_item in ConfigurationItem.objects.filter(shop=shop):
         configuration[conf_item.key] = conf_item.value
+    for conf_item in EncryptedConfigurationItem.objects.filter(shop=shop):
+        configuration[conf_item.key] = conf_item.value
```

Comment:
This will override the configurations with the same key that are not encrypted. Think you can add a prefix for encrypted items: `f"encrypted:{key}"`.
In the get() method also pass the `encrypted=False` flag so we can chose whether to get encrypted items or not.

Reply:
But the point is that only the encrypted value remains, right? So why do we keep the non encrypted value?